good-cheap-fast/script.js

```diff
@@ -0,0 +1,12 @@
+var elems = document.querySelectorAll("input[type='checkbox']");
```

Comment:
you can declare good, cheap fast separately. you can have more control over them since there are only three and are static.